### PR TITLE
change id refs to name

### PR DESCRIPTION
**Pull request by jainvipin**
making spdk bridge changes to move from id to name changes in opi api.
pls wait to merge after I remove refs to jainvipin/opi-api (go.mod), but submitting the PR to get review on the changes.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -22,3 +22,5 @@ require (
 	google.golang.org/genproto/googleapis/api v0.0.0-20230726155614-23370e0ffb3e // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230726155614-23370e0ffb3e // indirect
 )
+
+replace github.com/opiproject/opi-api v0.0.0-20230726170919-691a90a13429 => github.com/jainvipin/opi-api v0.0.0-20230727174915-645bea203166
```
Comment (glimchb) [inline go.mod:26]: please update to latest api repo commit, now that I merged your change

**Review by glimchb — CHANGES_REQUESTED** [review]
lgtm, please update to latest opi-api commit and latest godpu after that [PR](https://github.com/opiproject/godpu/pull/247) is merged

**Comment by jainvipin** [comment]
> lgtm, please update to latest opi-api commit and latest godpu after that [PR](https://github.com/opiproject/godpu/pull/247) is merged

done.

**Comment by glimchb** [comment]
> done.

please also update https://github.com/opiproject/opi-spdk-bridge/blob/main/docker-compose.yml#L71
to `c35775b11f5bafbdc3a24d5f7ad01de0ea31050592e040a3f55ec2b5911ee972` per https://github.com/opiproject/godpu/pkgs/container/godpu

**Comment by glimchb** [comment]
@jainvipin now fails on https://github.com/opiproject/opi-spdk-bridge/blob/main/scripts/tests.sh#L64-L81

**Review by glimchb — APPROVED** [review]
thank you @jainvipin for amazing help on breaking change implementation!

**Pull request merged**